packages/config/src/configDefaults.ts

```diff
@@ -146,7 +146,7 @@ export const configProps = ({
           config = {};
         }
 
-        config = _.extend({}, defaultTXValues, config);
+        config = assignIn({}, defaultTXValues, config);
```

Comment:
this change name?

Reply:
Yes, `extend` is an alias: https://lodash.com/docs/#assignIn